docs/source/tracking.rst

```diff
@@ -187,19 +187,60 @@ common tasks:
 Running a Tracking Server
 -------------------------
 
-The MLflow tracking server launched via ``mlflow ui`` also hosts REST APIs for tracking runs,
+The MLflow tracking server launched via ``mlflow server`` also hosts REST APIs for tracking runs,
 writing data to the local filesystem. You can specify a tracking server URI
 via the ``MLFLOW_TRACKING_URI`` environment variable and MLflow's tracking APIs will automatically
 communicate with the tracking server at that URI to create/get run information, log metrics, etc.
 
-For example, to launch a run against a local tracking server, launch ``mlflow ui``, set
-``MLFLOW_TRACKING_URI`` to ``http://localhost:5000``, and run:
+When running the remote server, it is recommended that you provide the `--artifact-root` command
+line option. This option will cause clients to log their artifact output (e.g., models) to this
+location which is suitable for large data (such as an S3 bucket). If you do not provide this option,
+then clients will write artifacts to `their` local directories, which the server can't serve.
+
+Remote server for development
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I think we capitalized other subtitles so do that here. Also see the note about shared FS -- it is a valid use case (especially in HPC and some enterprise environments) so just mention it has to be shared; it's not just for development.

Reply:
Actually I realized that this was "for development" because of the file store location. I think it makes more sense to just show the "for production" one directly and say that the file store defaults to ./mlruns, which is probably not the right place in many cases (and our example code can show passing another directory). It doesn't make a ton of sense to have two separate sections.

Reply:
The development server also serves HTTP to the world, which I didn't want to recommend for production.